docs/topics/testing/tools.txt

```diff
@@ -969,7 +969,7 @@ The code for this test may look as follows::
             super().tearDownClass()
 
         def test_login(self):
-            self.selenium.get('%s%s' % (self.live_server_url, '/login/'))
+            self.selenium.get(f'{self.live_server_url}/login/')
```

Comment:
since the Django code is formatted with black I would take advantage of these changes to update the formatting
```suggestion
            self.selenium.get(f"{self.live_server_url}/login/")
```

Reply:
I was trying to make these changes as minimal as possible, keeping the quote style consistent with the surrounding code.